pkg/helm/releases.go

```diff
@@ -37,3 +43,58 @@ func MakeHelmRelease(name, version, cluster, namespace string, helmRepository ty
 		},
 	}
 }
+
+// AppendHelmReleaseToString appends a HelmRelease to a string.
```

Comment:
:D Can you be a tiny bit more descriptive please? :) Why? How? What's the purpose of the append?